docs/development.md

```diff
@@ -129,7 +152,7 @@ A number of new environment variables have been setup to control parts of the bu
 
 | Name | Default | Description |
 | ----------- | --------- | ----------- |
-|GP2040_BOARDCONFIG |Pico |The boards.h config file to use for the build.|
+|GP2040_BOARDCONFIG |Pico |The boards. H config file to use for the build.|
```

Comment:
This is a legacy thing I think, should be 

"BoardConfig.h"

Reply:
Honestly, don't know why that was even changed